proxy/http/HttpTransact.cc

```diff
@@ -288,24 +288,26 @@ is_localhost(const char *name, int len)
   return (len == (sizeof(local) - 1)) && (memcmp(name, local, len) == 0);
 }
 
+/** Return whether the response code indicates an error response. */
 inline static bool
-is_response_simple_code(HTTPStatus response_code)
+is_response_error_code(HTTPStatus response_code)
```

Comment:
Is it theoretically valid for this to ever be different than `is_response_simple_code(response_code) || is_response_unavailable_code(response_code)` ?
If not, would it be better to make it that, to avoid duplication, one of them getting out of sync, etc?

Reply:
Due to their simplicity, I'd prefer to keep the semantic value of having a function that names the intention of the condition.